src/shared/components/query/QueryContainer.tsx

```diff
@@ -1,152 +1,277 @@
+										            {this.store.submitError}
+									            </span>
+                                            )}
+
+                                            {this.store.oqlMessages.map(msg => {
+                                                return (
+                                                    <span className={styles.oqlMessage}>
+												        <i className='fa fa-info-circle' style={{marginRight: 5}}/>
+                                                        {msg}
+											        </span>
+                                                );
+                                            })}
+                                        </FlexCol>
+                                    </FlexRow>
+                                </>
+                            </If>
+                            <If condition={!this.showQueryControls}>
+                                <FlexRow className={styles.forkedButtons}>
+                                    <div style={{width:330,textAlign:'right', marginRight:20}}>
+                                        <StudySelectorStats store={this.store}/>
+                                    </div>
+
+                                    <DefaultTooltip placement={"top"}
+                                                    trigger={["hover"]}
+                                                    destroyTooltipOnHide={true}
+                                                    mouseEnterDelay={0}
+                                                    mouseLeaveDelay={0}
+                                                    disabled={this.store.selectableSelectedStudies.length > 0}
+                                                    overlay={this.queryByGeneTooltipMessage}>
+                                        <a data-test="queryByGeneButton" onClick={()=>this.store.hasSelectedStudies && this.toggleQueryControls()}
+                                                className={classNames('btn btn-primary btn-lg', { disabled: !this.store.hasSelectedStudies } )}>
+                                            Query By Gene
+                                        </a>
+                                    </DefaultTooltip>
+                                    OR
+                                    <DefaultTooltip placement={"top"}
+                                                    trigger={["hover"]}
+                                                    destroyTooltipOnHide={true}
+                                                    mouseEnterDelay={0}
+                                                    mouseLeaveDelay={0}
+                                                    disabled={!this.exploreCohortsButtonDisabled}
+                                                    overlay={this.exploreCohortsButtonTooltipMessage}>
+                                            <a onClick={()=>!this.exploreCohortsButtonDisabled && this.store.openSummary()}
```

Comment:
another option (not 100% sure if it'll work) would be to wrap this <a> in a div which will thus receive the tooltip mouse events

Reply:
tried that but even with 10px padding around button it still doesn't always fire.